README.rst

```diff
@@ -1,4 +1,5 @@
 .. image:: https://github.com/zopefoundation/Products.DCWorkflow/workflows/tests/badge.svg
+        :target: https://github.com/zopefoundation/Products.DCWorkflow
```

Comment:
I was trying to find a direct link to a page that shows the test data like we have for Travis CI but didn't find one.

Reply:
Yeah, the URL looks ugly but it seems to work.